nossas/bonde
# Feature add community invitation

# Related issues
- Add page to invite community member #660

# Related pull-requests
- Feature add community invitation nossas/bonde-server#290

# How to test
- Log in into BONDE
- Choose a community
- In the mobilization list page, on the sidebar, the **Membros** item should exists
<img width="282" alt="screen shot 2017-06-13 at 16 45 50" src="https://user-images.githubusercontent.com/5435389/27101356-d062672a-5057-11e7-9f04-592c388573ac.png">

- Click on it and, it should redirect to `/community/invite` route
- It should render the community members page like screenshot below:
<img width="1283" alt="screen shot 2017-06-13 at 16 47 09" src="https://user-images.githubusercontent.com/5435389/27101391-f606bdaa-5057-11e7-8f14-33f89f1d8823.png">

- Validate the validations
  - Try to submit the form without entry anything
    - The error should be displayed at the **PESSOAS** input label:
<img width="1162" alt="screen shot 2017-06-13 at 16 48 43" src="https://user-images.githubusercontent.com/5435389/27101454-32c4243a-5058-11e7-9c51-047daace496d.png">

  - Try to entry an email that contains an invalid format
    - The error should be displayed like below:
<img width="1154" alt="screen shot 2017-06-13 at 16 52 56" src="https://user-images.githubusercontent.com/5435389/27101592-c70825f6-5058-11e7-96be-366f3d6e2117.png">

- If everything is ok, after clicking on **CONVIDAR** button or simply hitting the enter:
  - It should display the loader while requesting to the API
<img width="1632" alt="screen shot 2017-06-13 at 16 54 48" src="https://user-images.githubusercontent.com/5435389/27101660-09c84d80-5059-11e7-9d86-ab101db5d016.png">

  - It should display a success notification if the request to the API was a success:
<img width="391" alt="screen shot 2017-06-13 at 16 57 35" src="https://user-images.githubusercontent.com/5435389/27101779-6c6c922a-5059-11e7-928f-d33de36a7091.png">

  - Otherwise, a generic error notification should be displayed
<img width="398" alt="screen shot 2017-06-13 at 16 56 56" src="https://user-images.githubusercontent.com/5435389/27101749-55473618-5059-11e7-95c7-7ce3897bc9d2.png">
